src/types/emr/medicationStatement.ts

```diff
@@ -38,6 +49,11 @@ export type MedicationStatement = {
   information_source: MedicationStatementInformationSourceType;
 
   note?: string;
+
+  created_by: {
+    username: string;
+    profile_picture_url?: string;
+  };
```

Comment:
related to previous review....

this is already present with more appropriate types in `MedicationStatementRead` instead.